modules/microshift-greenboot-create-health-check-script.adoc

```diff
@@ -1,57 +1,114 @@
-= Creating a health check script
+:_content-type: CONCEPT
+[id="microshift-greenboot-app-health-check-script_{context}"]
+= How to create a health check script for your application
 
-You can create a health check script for installed workloads by placing them in the `/etc/greenboot/check/required.d` directory. The following procedure provides an example of installing the busybox application and creating a health check script for busybox. You can use this example as a general guide for creating health check scripts for your applications.
+You can create workload or application health check scripts in the text editor of your choice using the example in this documentation. Then save the scripts in the `/etc/greenboot/check/required.d` directory. When a script in the `/etc/greenboot/check/required.d` directory exits with an error, greenboot triggers a reboot in an attempt to heal the system.
 
-.Prerequisite
+[NOTE]
+====
+Any script placed in the `/etc/greenboot/check/required.d` directory will trigger a reboot if it exits with an error.
+====
 
-* You have installed a workload. For this example, the busybox application is used as a workload. The "Additional resources" section that follows this procedure has a link to instructions on deploying workloads using manifests.
+If your health check logic requires any post-check steps, you can also create additional scripts and save them in the relevant greenboot directories:
 
-.Procedure
+* You can also place shell scripts you want to run after a boot has been declared successful in `/etc/greenboot/green.d`.
+* You can place shell scripts you want to run after a boot has been declared failed in `/etc/greenboot/red.d`. For example, if you have steps to heal the system before restarting, you can create scripts for your use case and place them in the `/etc/greenboot/red.d` directory.
 
-. To create a health check script, run the following command:
-+
-[source, terminal]
-----
-$ SCRIPT_FILE=/etc/greenboot/check/required.d/50_busybox_running_check.sh
-sudo curl -s https://raw.githubusercontent.com/openshift/microshift/3b7f6025cd77bd1bf827416fd026783ead82b7c8/docs/config/busybox_running_check.sh \
-  -o ${SCRIPT_FILE} && echo SUCCESS || echo ERROR
-sudo chmod 755 ${SCRIPT_FILE}
-----
-+
-In this example, the script verifies that busybox is running as expected. You can replace `/etc/greenboot/check/required.d/50_busybox_running_check.sh` with your own workload details.
-+
-[NOTE]
+[id="microshift-greenboot-about-workload-health-check-script-example_{context}"]
+== About the workload health check script example
+
+The following example uses the {product-title} health check script as a template. You can use this example with the provided libraries as a guide for creating basic health check scripts for your applications.
+
+[id="microshift-greenboot-app-health-check-basic-prereqs_{context}"]
+=== Basic prerequisites for creating a health check script
```

Comment:
I haven't seen a "Basic prerequisites..." heading used in our docs before. I _think_ it is ok though.